doozerlib/rpmcfg.py

```diff
@@ -85,32 +84,73 @@ def push_tag(self):
             raise ValueError('Must run set_nvr() before calling!')
 
         with Dir(self.source_path):
-            exectools.cmd_assert('git push origin {}'.format(self.tag), retries=3)
+            exectools.cmd_assert('git push origin --tags', retries=3)
 
-    def commit_changes(self):
+    def commit_changes(self, scratch):
         with Dir(self.source_path):
+            if self.config.content.build.use_source_tito_config:
```

Comment:
curious if you should check if scratch mode here, as this would not be necessary

Reply:
tag is still needed in a scratch build (`tito release` looks at the tag); it just doesn't push the tag and commit.